chromium_src/chrome/browser/ui/views/tabs/tab.cc

```diff
@@ -20,3 +20,21 @@
 #include "src/chrome/browser/ui/views/tabs/tab.cc"
 #undef BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY
 #undef BRAVE_UI_VIEWS_TABS_TAB_ALERT_INDICATOR_POSITION
+
+// Introduced brave specific method because we moved alert button to left side
+// in the tab whereas upstream put it on right side. Need to consider this
+// change for calcuating largest selectable region.
+int Tab::GetWidthOfLargestSelectableRegionBrave() const {
+  // Assume the entire region except the area that alert indicator/close buttons
+  // occupied is available for click-to-select.
+  // If neither are visible, the entire tab region is available.
+  int selectable_width = width();
+  if (alert_indicator_button_->GetVisible()) {
+    selectable_width -= alert_indicator_button_->width();
+  }
+
+  if (close_button_->GetVisible())
+    selectable_width -= close_button_->width();
+
+  return selectable_width;
```

Comment:
do we need to add some clamping here to not accidentally return negative values here?

Reply:
I checked where this value is used at `AlertIndicatorButton` and it would be fine.
But, changed to  `return std::max(0, selectable_width);` for safe.

```
if (GetTab()->GetWidthOfLargestSelectableRegion() < required_width)
or
enable = GetTab()->GetWidthOfLargestSelectableRegion() >= required_width;
```